workspaces/fcopardo/app/src/main/kotlin/fcopardo/array/TwoSum.kt

```diff
@@ -0,0 +1,35 @@
+package app.src.main.kotlin.fcopardo.array
+
+class TwoSum {
```

Comment:
specify time and space complexity of your solution as a comment

Reply:
Also, update the coding list by marking an X at [this line](https://github.com/cfarfan/interview-preparation/blob/main/workspaces/fcopardo/coding-list.md?plain=1#L6) like this:
`- [X] [Two Sum](https://leetcode.com/problems/two-sum/)` ✌️